Content.Server/Nutrition/EntitySystems/SmokingSystem.cs

```diff
@@ -74,9 +71,15 @@ public void SetSmokableState(EntityUid uid, SmokableState state, SmokableCompone
             _items.SetHeldPrefix(uid, newState);
 
             if (state == SmokableState.Lit)
-                _active.Add(uid);
+                EnsureComp<BurningComponent>(uid);
             else
-                _active.Remove(uid);
+                RemComp<BurningComponent>(uid);
+
+            var sound = state == SmokableState.Lit
+                ? smokable.LightSound
+                : smokable.SnuffSound;
```

Comment:
```suggestion
            var sound = SoundSpecifier?;
            if (state == SmokableState.Lit)
            {
                EnsureComp<BurningComponent>(uid);
                sound = smokable.LightSound;
            }
            else
            {
                RemComp<BurningComponent>(uid);
                sound = smokable.SnuffSound;
            }
```
considering the if and ternary have the same condition.

Reply:
yeah, I like this more
or even just write `PlayPvs` twice so you can remove the `SoundSpecifier? sound = null;` line